src/scss/styles.scss

```diff
@@ -281,3 +281,29 @@ DD {
   font-size: .9em;
   color: #777;
 }
+
+#dropzone {
```

Comment:
Use classes for stylization, please. They allows to use modifiers like in other classes here. 
With modifier selectors will look like `.dropzone` and `.dropzone--active`